group-generators/helpers/data-providers/alchemy/index.ts

```diff
@@ -337,6 +338,39 @@ export class AlchemyProvider {
     }
   }
 
```

Comment:
I added a function to the Alchemy provider to get all the tokenId's from a contract.
I couldn't see a simple function to do that so I query the owners and filter the tokenids.

I used Alchemy rather than BigQuery because this can now be used for both ERC721 and 1155 - that would require 2 different updates for BigQuery.

Reply:
It wouldn't have been bad to have a second function for ERC721 tokens.
And It would be better to use BigQuery (instead of alchemy) for this case because networks are only Ethereum & Polygon.

Currently there not that much Parallel NFT, but if the number goes up, BigQuery would be way more scalable than Alchemy. But yes for now let's keeping what you did 👍